transformato/mutate.py

```diff
@@ -1221,8 +1221,8 @@ def _transform_common_core(self) -> list:
             logger.warning(f"Charge parameters mutation: {charge_mutation}.")
 
             t = CommonCoreTransformation(
-                self.get_common_core_idx_mol1() + self.dummy_region_cc1.lj_default,
-                self.get_common_core_idx_mol2() + self.dummy_region_cc2.lj_default,
+                self.get_common_core_idx_mol1(),
+                self.get_common_core_idx_mol2(),
                 self.psfs["m1"],
```

Comment:
I don't know if we can remove the `self.dummy_region_cc1.lj_default` variable, but in my opinion I think it's not necessary since this atom (the lj default atom) is not part of the common core anymore and thus can vary between different ligands